### PR TITLE
chore: SAP Cloud Application Event Hub and fixed tests

### PR DESCRIPTION
**Pull request by David-Kunz**

### CODE REVIEW

**Review by sjvans — APPROVED** [review]

**Comment by David-Kunz** [comment]
Hm, let me check why the tests are failing...

**Comment by David-Kunz** [comment]
Seems to be caused by some update in @sap/cds.

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "test": "npx jest --silent"
   },
   "peerDependencies": {
-    "@sap/cds": ">=7"
+    "@sap/cds": "*"
```
Comment (sjvans) [inline package.json:21]:
```suggestion
    "@sap/cds": ">=7"
```

Comment (David-Kunz) [reply package.json:21]: I don't like having that. In the worst case, @sap/cds is installed twice. I'd rather have a version check in the plugin itself.

Comment (David-Kunz) [reply package.json]: reverted back

**Review by sjvans — APPROVED** [review]

**Pull request merged**